projects/graph/src/graph.py

```diff
@@ -6,4 +6,12 @@
 class Graph:
     """Represent a graph as a dictionary of vertices mapping labels to edges."""
     def __init__(self):
-        pass  # TODO
+        self.vertices = dict()
+    def add_vertex(self,vertex):
+        self.vertices[vertex] = set()
```

Comment:
You may want to add a conditional here to raise an Exception if the vertex already exists, or else you might end up with overlapping duplicates.

Reply:
I don't believe python allows duplicate keys in a dictionary